test/unit/im21doorTest.js

```diff
@@ -0,0 +1,69 @@
+const { expect } = require('chai')
+const { describe, it } = require('mocha')
+const im21door = require('../../im21door')
+
+describe('im21door.js unit tests', () => {
+  describe('isLowBattery', () => {
+    it('should return true if and only if bit-2 is 1', () => {
+      const testCases = [
+        { inputSignal: '00', expected: false },
+        { inputSignal: '01', expected: false },
+        { inputSignal: '02', expected: false },
+        { inputSignal: '03', expected: false },
+        { inputSignal: '04', expected: true },
+        { inputSignal: '05', expected: true },
+        { inputSignal: '06', expected: true },
+        { inputSignal: '07', expected: true },
+        { inputSignal: '08', expected: false },
+        { inputSignal: '09', expected: false },
+        { inputSignal: '0A', expected: false },
+        { inputSignal: '0B', expected: false },
+        { inputSignal: '0C', expected: true },
+        { inputSignal: '0D', expected: true },
+        { inputSignal: '0E', expected: true },
+        { inputSignal: '0F', expected: true },
+      ]
+      expect(testCases.map(testCase => `${testCase.inputSignal}: ${im21door.isLowBattery(testCase.inputSignal)}`)).to.eql(
```

Comment:
Neat!

Reply:
Yeah, I didn't want to write them all out as separate `it` blocks, lol.